4001-5000/LIT4975Dersan.xml

```diff
@@ -4,9 +4,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘደረሰ፡ ቅዱስ፡ ጌርሎስ፡ ሊቀ፡ ጳጳሳት፡ ዘኢየሩሳለም፡</title>
-                <title xml:lang="gez" type="normalized" corresp="#1">Dǝrsān za-darasa qǝddus Gerlos liqa ṗaṗasāt za-ʾIyarusālem</title>
-                <title xml:lang="eng" corresp="#1">Homily pronounced by St Cyril, bishop of Jerusalem</title>
+                <title xml:id="t1" xml:lang="en">Homily on the Presentation of Jesus</title>
+                <title xml:lang="gez" xml:id="t2">ድርሳን፡ ዘደረሰ፡ ቅዱስ፡ ጌርሎስ፡ ሊቀ፡ ጳጳሳት፡ ዘኢየሩሳለም፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t2">Dǝrsān za-darasa qǝddus Gerlos liqa ṗaṗasāt za-ʾIyarusālem</title>
```

Comment:
ṗaṗṗasāt?

Reply:
I think we write it with one ṗ.

Reply:
yes :) https://betamasaheft.eu/Guidelines/?id=dubious-spelling